src/Microsoft.ManagedZlib/tests/MyClassTests.cs

```diff
@@ -0,0 +1,73 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace Microsoft.ManagedZLib.Tests
+{
+    public class MyClassTests
+    {
+        public Stream CreateStream(Stream stream, CompressionMode mode) => new DeflateStream(stream, mode);//For test1
+        public Stream CreateStream(Stream stream, CompressionMode mode, bool leaveOpen) => new DeflateStream(stream, mode, leaveOpen);
+        public Stream CreateStream(Stream stream, CompressionLevel level) => new DeflateStream(stream, level);
+        public Stream CreateStream(Stream stream, CompressionLevel level, bool leaveOpen) => new DeflateStream(stream, level, leaveOpen);
+        public Stream BaseStream(Stream stream) => ((DeflateStream)stream).BaseStream;
+        protected string CompressedTestFile(string uncompressedPath) => Path.Combine("DeflateTestData", Path.GetFileName(uncompressedPath));
+        
+        public static IEnumerable<object[]> UncompressedTestFiles()
+        {
+            yield return new object[] { Path.Combine("UncompressedTestFiles", "TestDocument.doc") };
+            //yield return new object[] { Path.Combine("UncompressedTestFiles", "TestDocument.docx") };
+            //yield return new object[] { Path.Combine("UncompressedTestFiles", "TestDocument.pdf") };
+            //yield return new object[] { Path.Combine("UncompressedTestFiles", "TestDocument.txt") };
+            //yield return new object[] { Path.Combine("UncompressedTestFiles", "alice29.txt") };
+            //yield return new object[] { Path.Combine("UncompressedTestFiles", "asyoulik.txt") };
+            //yield return new object[] { Path.Combine("UncompressedTestFiles", "cp.html") };
+            //yield return new object[] { Path.Combine("UncompressedTestFiles", "fields.c") };
+            //yield return new object[] { Path.Combine("UncompressedTestFiles", "grammar.lsp") };
+            //yield return new object[] { Path.Combine("UncompressedTestFiles", "kennedy.xls") };
+            //yield return new object[] { Path.Combine("UncompressedTestFiles", "lcet10.txt") };
+            //yield return new object[] { Path.Combine("UncompressedTestFiles", "plrabn12.txt") };
+            //yield return new object[] { Path.Combine("UncompressedTestFiles", "ptt5") };
+            //yield return new object[] { Path.Combine("UncompressedTestFiles", "sum") };
+            //yield return new object[] { Path.Combine("UncompressedTestFiles", "xargs.1") };
```

Comment:
As I mentioned yesterday, you might have an easier way debugging if you use `InlineData` instead o `MemberData` in your unit test. That way, in VS, you can choose an individual data item to use, which you cannot do when using `MemberData` (it runs all).